deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -126,17 +126,36 @@ After the benchmark has been run, the indices and documents of the source and ta
+2. The access policy on the Domain should be an open access policy that allows all access or an access policy that at least allows the IAM task roles for the applicable Migration services (Traffic Replayer, Migration Console, Fetch Migration)
 
 #### OpenSearch Serverless
-When using an existing OpenSearch Serverless Collection, some additional configuration is needed for the data access policies to allow the Traffic Replayer and Migration Console services to communicate with the Serverless cluster. Mainly, the IAM task roles for these Migration services should be added to a data access policy that allows them permission to perform all [index operations](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-data-access.html#serverless-data-supported-permissions) (`aoss:*`) for all indexes in the given collection. This will allow the Replayer to create, modify, and delete indexes as it mirrors incoming traffic. This can also be focused down to specific indexes or operations for sensitive cases, but should be monitored as this could prevent some requests from being replayed. 
+A Collection, will need to configure a Network and Data Access policy to allow proper functioning of this solution
+1. The Collection should have a network policy that has a `VPC` access type by creating a VPC endpoint on the VPC used for this solution. This VPC endpoint should be configured for the private subnets of the VPC and attach the `osClusterAccessSG` security group.
+2. The data access policy needed should grant permission to perform all [index operations](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-data-access.html#serverless-data-supported-permissions) (`aoss:*`) for all indexes in the given collection, and use the task roles of the applicable Migration services (Traffic Replayer, Migration Console, Fetch Migration) as the principals for this data access policy. This could potentially be focused down to specific indexes or operations for sensitive cases, but should be monitored as this could prevent some historical-data/requests from being replayed to the Collection
 
+See [Configuring SigV4 Replayer Requests](#configuring-sigv4-replayer-requests) for details on enabling SigV4 requests from the Traffic Replayer to the target cluster after following the previous setup
+
+## Configuring SigV4 Replayer Requests
+With any [required setup](#importing-target-clusters) on the target cluster having been completed, a user can then use the `trafficReplayerExtraArgs` option to specify the Traffic Replayer argument for enabling SigV4 auth, which the below sections show. **Note**: The `trafficReplayerEnableClusterFGACAuth` option should not be used if enabling SigV4 auth for the Traffic Replayer as only one auth mechanism should be specified. See [here](#how-is-an-authorization-header-set-for-requests-from-the-replayer-to-the-target-cluster) for more details on how the Traffic Replayer sets its auth header.
```

Comment:
This is confusing.  I think that dropping the second word, "any" for "the" helps simplify it.
The trafficReplayerExtraArgs are args to what?  
"as only one auth mechanism should be specified"... Rotating/splitting this sentence might help.... Only one auth (authN or authZ?) mechanism, FGAC/SigV4..., can be used.  
Is SigV4 referred to as SigV4 or AWS Auth in the service documentation?

Reply:
Args for the Traffic Replayer, if this is not clear let me know what can help.  

Have rearranged the structure. I've updated to be more precise here but it seems like a combo of wanting to use SigV4 authentication for our authorization header. Let me know if you have better wording.

I've normally seen SigV4: https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html